src/analysis/Environments/Environments.ts

```diff
@@ -781,11 +789,12 @@ export const Environments = (props: EnvironmentsProps): ReactNode => {
                 headerRenderer: () =>
                   h(Sortable, { sort: diskSort, field: 'workspace', onSort: setDiskSort }, ['Workspace']),
                 cellRenderer: ({ rowIndex }) => {
-                  const { status: diskStatus, googleProject, workspace, creator } = filteredDisks[rowIndex];
+                  const rowDisk = filteredDisks[rowIndex];
+                  const { status: diskStatus, cloudContext, workspace } = rowDisk;
```

Comment:
can we pull the creator off the audit info here? I believe the downstream component has that as optional (in the logic, not with sensible types to indicate such), so currently it just isn't displayed, but the info is there for all disks. It's just not being extracted properly.

Reply:
I decided to have the implementation of permissions.canDeleteDisk be where poking into audit info happens.  So that that concern is removed from the component here.

Reply:
Ah sorry, I thought this was for a different column at first (the details view column), which displays this field properly.